music/src/core/player.ts

```diff
@@ -212,25 +221,36 @@ export abstract class BasePlayer {
   }
 
   /**
-   * Pause playing the currently playing sequence right away. Call unpause()
+   * Pause playing the currently playing sequence right away. Call resume()
    * to resume.
+   * @throws {Error} If the player is stopped.
    */
   pause() {
+    if (!this.isPlaying()) {
+      throw new Error('Cannot pause playback while the player is stopped.');
+    }
     Tone.Transport.pause();
   }
 
   /**
-   * Pause playing the currently playing sequence right away. Call resume()
-   * to resume playing the sequence.
+   * Resume playing the sequence after pause().
```

Comment:
Wow, thanks for fixing these comments. I don't know how I shipped them like this 🤦‍♀